mobile/cmp/dimensionchooser/DimensionChooser.js

```diff
@@ -34,7 +34,10 @@ export class DimensionChooser extends Component {
         buttonWidth: PT.number,
 
         /** Width in pixels of the popover menu itself. */
-        dialogWidth: PT.number
+        dialogWidth: PT.number,
+
+        /** Primary component model instance. */
+        model: PT.oneOfType([PT.instanceOf(DimensionChooserModel), PT.object]).isRequired
```

Comment:
Unlike with colChooser, the developer _will_ need to create this component and pass it a model, since it's not rendered directly by grid.  So good to have the prop-type in place, although this is another instance where we only want the class form - not the object.  Idea again here is that an owning/higher-level component would create the model, pass it to this comp, and observe its value to detect changes to dimensions made by the user interacting with this comp.